dashboard/src/components/Accordion/Accordion.tsx

```diff
@@ -141,29 +160,68 @@ const AccordionBuildsTrigger = ({
 const AccordionTestsTrigger = ({
   accordionData,
 }: IAccordionItems): JSX.Element => {
-  const triggerInfo = accordionData as AccordionItemTestsTrigger;
+  const triggerInfo = accordionData as TPathTests;
   return (
     <>
-      <TableCell>{triggerInfo.testPlans}</TableCell>
+      <TableCell>{triggerInfo.path_group}</TableCell>
       <TableCell className="flex flex-row gap-1">
-        <ColoredCircle
-          quantity={triggerInfo.testErrors ?? 0}
-          backgroundClassName={ItemType.Error}
-        />
-        <ColoredCircle
-          quantity={triggerInfo.testSuccessfull ?? 0}
-          backgroundClassName={ItemType.Warning}
+        <TestStatus
```

Comment:
add a cursor: pointer here and a hover that shows the user that it is a distinc click (maybe a filter brightnesss (0.6) )